solr/modules/ltr/src/test/org/apache/solr/ltr/TestRerankBase.java

```diff
@@ -134,40 +134,48 @@ public static ManagedModelStore getManagedModelStore() {
 
   protected static void setupTestInit(String solrconfig, String schema, boolean isPersistent)
       throws Exception {
-    tmpSolrHome = createTempDir().toFile();
-    tmpConfDir = new File(tmpSolrHome, CONF_DIR);
-    tmpConfDir.deleteOnExit();
-    FileUtils.copyDirectory(new File(TEST_HOME()), tmpSolrHome.getAbsoluteFile());
+    tmpSolrHome = createTempDir();
+    tmpConfDir = tmpSolrHome.resolve(CONF_DIR);
+    tmpConfDir.toFile().deleteOnExit();
+    PathUtils.copyDirectory(Path.of(TEST_HOME()), tmpSolrHome.toAbsolutePath());
```

Comment:
`TEST_PATH()`

Reply:
Fixed in e2c5a83f779648f50b512743577f40f6976cede1